client/src/components/pages/admin/Submitted_all/index.js

```diff
@@ -1,5 +1,68 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import MaterialTable from 'material-table';
+import { Helmet } from 'react-helmet';
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+import backGround from '../../../../assets/images/backgroundDash.svg';
+import DashBar from '../../../dashboard/Tabs';
 
-const SubmittedAll = () => <h1>SubmittedAll</h1>;
+import './index.css';
 
-export default SubmittedAll;
+const getSubmitted = async () => {
+  const { data } = (await axios.get('/api/v1/dashboard/applicants')).data;
+  return data.filter((element) => {
+    if (element.applicationSubmittedDate) return element;
+  });
+};
+
+const Header = {
+  columns: [
+    { title: 'Name', field: 'fullName', width: '20%' },
+    { title: 'Email', field: 'email', width: '20%' },
+    { title: 'Mobile', field: 'mobileNumber' },
+    { title: 'Age', field: 'age' },
+    { title: 'Address', field: 'address' },
```

Comment:
for Academy place Gaza, Khalil use 
```suggestion
    { title: 'Academy Location', field: 'location' },
    { title: 'Address', field: 'address' },
```